src/index.css

```diff
@@ -0,0 +1,89 @@
+@import url('https://fonts.googleapis.com/css2?family=Abel&family=Dancing+Script:wght@400..700&family=Inter:ital,opsz,wght@0,14..32,100..900;1,14..32,100..900&family=Playwrite+CU:wght@100..400&family=Playwrite+PE:wght@100..400&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap');
+
+.auth-container {
+    background-image: url(./assets/images/authBg.png);
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+}
+.auth-bg {
+    background-color: #315951E5;
```

Comment:
We need to avoid having fixed colors and use css variables instead, as we will need to implement light/dark mode feature. Having fixed colors will make this very difficult to implement.